src/Microsoft.Framework.CommandLineUtils.Sources/CommandLine/CommandLineApplication.cs

```diff
@@ -432,6 +428,23 @@ public void ShowVersion()
             Console.WriteLine(VersionGetter());
         }
 
+        public string GetFullNameAndVersion()
+        {
+            return VersionGetter == null ? FullName : string.Format("{0} v{1}", FullName, VersionGetter());
```

Comment:
This file will be compiled by Mono msc compiler, which doesn't support string interpolation. So using `string.Format()` here